### DISCUSSION TITLE
9歲吉娃娃爆瘦 AI尿檢找到病因

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS3073-C/Python-Community-GS3073-C#107] [General]
文章:https://www.chinatimes.com/realtimenews/20211105001840-260405?chdtv
透過AI科技讓原本由人力需要做很久的東西，變得非常快速且方便，且由此可見AI科技已經致力於運用在醫療結合，感覺真的可以透過拍自己的便便來檢測自己有沒有生病，可是發明這個的人可能要到處蒐集便便，到醫院廁所跟生病的人說借我你便便的肖像權之類的。

**Comment by chia-kai-chang** [comment]
從文章的內文敘述，我認為這是AI的效率價值。

**Comment by briankuo123** [comment]
想到別人會有我尿或便便的數據還是蠻害羞的，不過要是AI能幫助醫療科技變得越來越發達，那將會是一件值得高興的事情。

**Comment by qeeeee39** [comment]
等這個便便檢測功能強大以後，人人都可以在家自己檢測，超方便

**Comment by ghost** [comment]
挨家挨戶地拍攝排泄物感覺是一件勞累的事 而且好害羞XD

**Comment by handsomeniko** [comment]
確實AI的應用展現了其效率價值，節省很督多領域的工作時間。

**Comment by Shiba010** [comment]
這項技術可以隨時透過排泄物偵測自家寵物的將康狀況，非常實用！
